packages/perseus-editor/src/components/section-control-button.tsx

```diff
@@ -1,53 +1,36 @@
-import {components} from "@khanacademy/perseus";
-import Clickable from "@khanacademy/wonder-blocks-clickable";
-import {color} from "@khanacademy/wonder-blocks-tokens";
+import IconButton from "@khanacademy/wonder-blocks-icon-button";
 import {StyleSheet} from "aphrodite";
 import * as React from "react";
 
-import type {PropsFor} from "@khanacademy/wonder-blocks-core";
-
-const {InlineIcon} = components;
-
-type InlineIconProps = PropsFor<typeof InlineIcon>;
-
 type SectionControlButtonProps = {
-    icon: InlineIconProps;
+    icon: any;
```

Comment:
Can this be more specific than `any`?  Perhaps its type could be `PropsFor<typeof IconButton>["icon"]`